core/flyout_base.js

```diff
@@ -264,6 +264,14 @@ Blockly.Flyout.prototype.dragAngleRange_ = 70;
  */
 Blockly.Flyout.prototype.scrollAnimationFraction = 0.3;
 
+/**
+ * Whether to recycle blocks when refreshing the flyout. Nothing can be
```

Comment:
I think this comment needs some clarity, because right now it seems like the second sentence is referring to the initial state of this property, when really it's referring to the fact that this initial value can't be used when the locale changes. I would just take the locale part of it out since it describes a use case for changing this property rather than a description of what this property itself does.